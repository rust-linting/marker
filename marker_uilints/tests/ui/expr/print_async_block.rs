fn something(x: &u8) {
    let _print_async_block = async move { *x };
}
