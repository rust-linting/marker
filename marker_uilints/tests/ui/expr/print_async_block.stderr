warning: print test
 --> $DIR/print_async_block.rs:2:5
  |
2 |     let _print_async_block = async move { *x };
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: Block(
              BlockExpr {
                  data: CommonExprData {
                      _lifetime: PhantomData<&()>,
                      id: ExprId(..),
                      span: SpanId(..),
                  },
                  stmts: [],
                  expr: Some(
                      UnaryOp(
                          UnaryOpExpr {
                              data: CommonExprData {
                                  _lifetime: PhantomData<&()>,
                                  id: ExprId(..),
                                  span: SpanId(..),
                              },
                              expr: Path(
                                  PathExpr {
                                      data: CommonExprData {
                                          _lifetime: PhantomData<&()>,
                                          id: ExprId(..),
                                          span: SpanId(..),
                                      },
                                      path: AstQPath {
                                          self_ty: None,
                                          path_ty: None,
                                          path: AstPath {
                                              segments: [
                                                  AstPathSegment {
                                                      ident: Ident {
                                                          name: "x",
                                                          span: $DIR/print_async_block.rs:2:44 - 2:45,
                                                      },
                                                      generics: SynGenericArgs {
                                                          args: [],
                                                      },
                                                  },
                                              ],
                                          },
                                          target: Var(
                                              VarId(..),
                                          ),
                                      },
                                  },
                              ),
                              kind: Deref,
                          },
                      ),
                  ),
                  label: None,
                  safety: Safe,
                  syncness: Async,
                  capture_kind: Move,
              },
          )
  = note: `#[warn(marker::test_lint)]` on by default

warning: 1 warning emitted

